### PR TITLE
add test-private-ui image bacl

### PR DESCRIPTION
**Pull request by yapei**
add test-private-ui image back

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml [hunk 1]
```diff
@@ -29,6 +29,15 @@ images:
...
   to: tests-private
+- dockerfile_literal: |
+    FROM src
+    RUN curl --silent --location https://rpm.nodesource.com/setup_lts.x | bash - && \
+        curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
+        PACKAGES="openssh-clients httpd-tools nodejs yarn xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib" && \
+        yum install --setopt=tsflags=nodocs -y $PACKAGES && \
+        yum clean all && rm -rf /var/cache/yum/*
+  from: src
+  to: tests-private-ui
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml:40]: @yapei you miss
```yaml
excluded_images:
  - tests-private-ui
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10.yaml [hunk 2]
```diff
@@ -25,6 +25,15 @@ images:
...
   to: tests-private
+- dockerfile_literal: |
+    FROM src
+    RUN curl --silent --location https://rpm.nodesource.com/setup_lts.x | bash - && \
+        curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
+        PACKAGES="openssh-clients httpd-tools nodejs yarn xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib" && \
+        yum install --setopt=tsflags=nodocs -y $PACKAGES && \
+        yum clean all && rm -rf /var/cache/yum/*
+  from: src
+  to: tests-private-ui
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10.yaml:36]: @yapei same comment.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11.yaml [hunk 3]
```diff
@@ -29,6 +29,15 @@ images:
...
   to: tests-private
+- dockerfile_literal: |
+    FROM src
+    RUN curl --silent --location https://rpm.nodesource.com/setup_lts.x | bash - && \
+        curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
+        PACKAGES="openssh-clients httpd-tools nodejs yarn xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib" && \
+        yum install --setopt=tsflags=nodocs -y $PACKAGES && \
+        yum clean all && rm -rf /var/cache/yum/*
+  from: src
+  to: tests-private-ui
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11.yaml:40]: @yapei same comment.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12.yaml [hunk 4]
```diff
@@ -29,6 +29,15 @@ images:
...
   to: tests-private
+- dockerfile_literal: |
+    FROM src
+    RUN curl --silent --location https://rpm.nodesource.com/setup_lts.x | bash - && \
+        curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
+        PACKAGES="openssh-clients httpd-tools nodejs yarn xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib" && \
+        yum install --setopt=tsflags=nodocs -y $PACKAGES && \
+        yum clean all && rm -rf /var/cache/yum/*
+  from: src
+  to: tests-private-ui
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12.yaml:40]: @yapei same comment.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13.yaml [hunk 5]
```diff
@@ -29,6 +29,15 @@ images:
...
   to: tests-private
+- dockerfile_literal: |
+    FROM src
+    RUN curl --silent --location https://rpm.nodesource.com/setup_lts.x | bash - && \
+        curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
+        PACKAGES="openssh-clients httpd-tools nodejs yarn xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib" && \
+        yum install --setopt=tsflags=nodocs -y $PACKAGES && \
+        yum clean all && rm -rf /var/cache/yum/*
+  from: src
+  to: tests-private-ui
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13.yaml:40]: @yapei same comment.

**Comment by kuiwang02** [comment]
@yapei after you change the code per comment, please ensure to execute `make update` before submitting code.

**Comment by yapei** [comment]
it looks like `release-4.13/images-images` job is failing more frequently due to 
```
{  the build tests-private failed after 6m44s with reason DockerBuildFailed: Dockerfile build strategy has failed.}
```

and `debug-disasterrecovery-aws-ipi`  job is failing frequently due to 
```
------------------the case selected------------------
No Case Selected
do not force success exit
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-08-23T10:44:09Z"}
error: failed to execute wrapped command: exit status 1
}
```

**Pull request merged**